README.markdown

```diff
@@ -170,6 +170,10 @@ new **Server**()
   * register a given resource for the API.  the appropriate REST
     verbs will be routed.
 
+**static**(*path*)
+
+  * add a static path to the server for optional static contents(demo html, assets)
```

Comment:
Please limit the line length to 70 characters here.  It's arbitrary but it's how the rest of the README is formatted.